wrappers/golang/curves/bls12377/ecntt.go

```diff
@@ -0,0 +1,41 @@
+//go:build ecntt
```

Comment:
I think its best to remove the build tags for g2 and ecntt and move these functionalities to separate packages underneath each curve package…apparently its an "abuse" of build tags to add API functionality and we didn’t know this when we first added G2; see [this comment](https://github.com/golang/go/issues/43288#issuecomment-749152792) and discussion